packages/crypto-wallet-core/src/transactions/doge/index.ts

```diff
@@ -13,7 +13,13 @@ export class DOGETxProvider extends BTCTxProvider {
       });
       return new this.lib.Transaction.UnspentOutput(btcUtxo);
     });
-    let tx = new this.lib.Transaction().from(btcUtxos).feePerByte(Number(fee) + 2);
+    let tx = new this.lib.Transaction().from(btcUtxos).feePerKb(Number(fee) + 2);
+    if (fee) {
+      tx.fee(fee);
+    }
+    if (feeRate) {
+      tx.feePerKb(Number(feeRate) / 1000); // feeRate is in feePerByte
```

Comment:
if feeRate is feePerByte, the feePerKb should be  `feePerKb  = feeRate * 1000` right?